packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -435,3 +428,22 @@ def _get_enum_class_name(name: str) -> str:
     else:
         name = name[0].upper() + name[1:]
     return f"{name}Enum"
+
+
+def _validate_and_transform_enum_annotations(enum_annotations: str) -> str:
+    enum_values = dict(json.loads(enum_annotations))
+    transformed_enum_annotations = {}
+    for enum_value, value in enum_values.items():
+        original_enum_value = enum_value
+
+        enum_value = re.sub(r"\W+", "_", enum_value).replace(" ", "_")
```

Comment:
`\W` already includes whitespace.
```suggestion
        enum_value = re.sub(r"\W+", "_", enum_value)
```